specification/communication/data-plane/CallingServer/preview/2021-08-30-preview/communicationservicescallingserver.json

```
@@ -0,0 +1,1503 @@
+        }
+      }
+    },
+    "ResultInfo": {
+      "description": "Result info class to be used to report result status for actions/operations.",
+      "required": [
+        "code",
+        "subcode"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "format": "int32",
+          "description": "The result code associated with the operation.",
+          "type": "integer"
+        },
+        "subcode": {
+          "format": "int32",
+          "description": "The subcode that further classifies the result.",
+          "type": "integer"
+        },
+        "message": {
+          "description": "The message is a detail explanation of subcode.",
+          "type": "string"
+        }
+      }
+    },
+    "CancelAllMediaOperationsRequest": {
+      "description": "The request payload for cancel all media operations",
+      "type": "object",
+      "properties": {
+        "operationContext": {
+          "description": "The context for this operation.",
+          "type": "string"
+        }
+      }
+    },
+    "CancelAllMediaOperationsResult": {
+      "description": "The response payload of the cancel all media operations.",
+      "required": [
+        "status"
+      ],
+      "type": "object",
+      "properties": {
+        "operationId": {
+          "description": "The operation id.",
+          "type": "string"
+        },
+        "status": {
+          "$ref": "#/definitions/OperationStatus"
+        },
+        "operationContext": {
+          "description": "The operation context provided by client.",
+          "type": "string"
+        },
+        "resultInfo": {
+          "$ref": "#/definitions/ResultInfo",
+          "description": "The result info for the operation."
+        }
+      }
+    },
+    "AddParticipantRequest": {
+      "description": "The add participant request.",
+      "type": "object",
+      "properties": {
+        "alternateCallerId": {
+          "$ref": "#/definitions/PhoneNumberIdentifier",
+          "description": "The alternate identity of source participant."
+        },
+        "participant": {
+          "$ref": "#/definitions/CommunicationIdentifier",
+          "description": "The participant to be added to the call."
+        },
+        "operationContext": {
+          "description": "The operation context.",
+          "type": "string"
+        },
+        "callbackUri": {
+          "description": "The callback URI.",
+          "type": "string"
+        }
+      }
+    },
+    "AddParticipantResult": {
+      "description": "The add participant result",
+      "type": "object",
+      "properties": {
+        "participantId": {
+          "description": "The id of the added participant.",
+          "type": "string"
+        }
+      }
+    },
+    "RecordingContentType": {
+      "description": "Gets or sets the content type of call recording.",
+      "enum": [
+        "audio",
+        "audioVideo"
```

In our phone numbers API, we've used + for combinations, so my weak preference would be 
```suggestion
        "audio+video"
```

---

Hey @DominikMe We have been using "AudioVideo" enum value for channelType everywhere in Recording service, internal OC/TC sdk. If I change this value I should change that everywhere to keep it consistent. What do you think ? or "AudioVideo" value sounds good ?

---

I'm okay with either and don't feel too strongly about this. The other Azure services aren't very consistent on the format in their enums either.